style.css

```diff
@@ -0,0 +1,220 @@
+.header {
+  height: 48px;
+  width: 375px;
+  border-radius: 0;
+  display: flex;
+  background: #3c3a39;
+  justify-content: space-between;
+}
+
+.mylogo {
+  background: #fff;
+  width: 70px;
+  height: 24px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.placeholder {
+  padding: 10px 12px;
+  position: absolute;
+  width: 94px;
+  height: 44px;
+  left: 12px;
+  top: 2px;
+  border-radius: 0;
+}
+
+.hamburger {
+  position: absolute;
+  height: 40px;
+  width: 40px;
+  left: 311px;
+  top: 4px;
+  padding: 8px;
+  border-radius: 8px;
+  flex-direction: row;
+  align-items: flex-start;
+  right: 24px;
+}
+
+.headline {
+  height: 590px;
+  width: 375px;
+  background: #1c1a19;
+}
+
+.primarytext {
+  height: 124px;
+  width: 327px;
+  left: 24px;
+  top: 105px;
+  font-family: Crete Round, serif;
+  font-size: 30px;
+  font-weight: 140;
+  line-height: 52px;
+  letter-spacing: 0.37px;
+  text-align: left;
+  color: #ff6b00;
+  position: absolute;
+}
+
+.supporting {
+  height: 27px;
+  width: 327px;
+  left: 24px;
+  top: 230px;
+  font-family: Roboto, sans-serif;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #fff5e1;
+  position: absolute;
+}
+
+.paragraph {
+  height: 123px;
+  width: 327px;
+  left: 24px;
+  top: 275px;
+  font-family: Poppins, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #fff;
+  position: absolute;
+}
+
+.social {
+  left: 24px;
+  top: 397px;
+  border-radius: 0;
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  width: 200px;
+  height: 40px;
+  justify-content: space-evenly;
+}
```

Comment:
- [ ] Although you did an impressive job designing the first page of your portfolio :+1:, to meet the Figma template requirements, you need to add more space between the paragraph and social links. This will make it easier for users to distinguish between the two sections and provide a better user experience.

**Your design**

![Screenshot from 2023-04-26 08-02-52](https://user-images.githubusercontent.com/105117832/234584073-15ce2efc-4025-4dfe-b089-3305a3f8e50c.png)

**Figma**

![Screenshot from 2023-04-26 08-02-39](https://user-images.githubusercontent.com/105117832/234584132-5c359864-dce4-4a19-9d3d-f6b8ca87a5c0.png)